scripts/globals/garrison.lua

```diff
@@ -115,9 +115,8 @@ xi.garrison.spawnNPC = function(zone, zoneData, x, y, z, rot, name, look)
         rotation = rot,
         look = look,
 
-        -- TODO: Make relevant group and pool entries for NPCs
-        groupId = 35,
-        groupZoneId = xi.zone.NORTH_GUSTABERG,
+        groupId = 1,
```

Comment:
There isn't a group 1 in GM_Home (210), what would this do?

Reply:
oh in ASB there is one (I am using what was there) for Garrison npcs